xbmc/pvr/channels/PVRChannelsPath.h

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright (C) 2012-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <string>
+
+class CDateTime;
+
+namespace PVR
+{
+  class CPVRChannelsPath
```

Comment:
> Do not indent namespaces to simplify nesting them and wrapping .cpp files in a namespace.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation

Reply:
I'd like to discuss this rule. Every "do not indent next line after an opening  curly brace" looks ugly and unnatural to me.